include/votca/xtp/orbitals.h

```diff
@@ -346,6 +346,13 @@ class Orbitals {
   bool GetFlagUseHqpOffdiag() const { return _use_Hqp_offdiag; };
   void SetFlagUseHqpOffdiag(bool flag) { _use_Hqp_offdiag = flag; };
```

Comment:
I am not convinced that adding this to orbitals is a good idea, because it has not enough information like this to really checkpoint from it.

Reply:
Nope. Did not add these lines. Felipe did. It isn't a part of this project.

Reply:
My fault, I am talking about the PMLocalizedOrbitals()